src/SessionManagementClient.ts

```diff
@@ -33,7 +33,7 @@ export class SessionManagementClient implements SessionManagementAPI {
     }
 
     getDomain(): string {
-        return this.matrixClient.getDomain()
+        return this.matrixClient.getDomain() || 'algundefaultDomain?'
```

Comment:
Should we ensure the domain is present?

Reply:
i thought about setting a default as it's only used [here](https://github.com/decentraland/unity-renderer/blob/00e78b1cc67c13de424e12dce1e9c877bc7993c9/browser-interface/packages/shared/friends/utils.ts#L36). and we're already setting a default for the cases where the client is empty.